### ISSUE TITLE
pretrain and fine-tune

### ISSUE THREAD

**Issue by zh-song** [lyuwenyu/RT-DETR#75]
how can i load the pretrained model based on coco to train on my dataset

**Comment by lyuwenyu** [comment]
Yes.  please reference this issue https://github.com/lyuwenyu/RT-DETR/issues/81

**Comment by lyuwenyu** [comment]
I add tuning code just now, you can try it by add `-t path/to/checkpoint`. https://github.com/lyuwenyu/RT-DETR/commit/64836918370ef3addf15e88c625300d57828105a
